src/js/osweb/items/inline_script.js

```diff
@@ -56,54 +56,6 @@ export default class InlineScript extends Item {
     this.vars._run = ''
   }
 
-  /**

```

Comment:
inline_script is not supported anyway, right? Should we leave this file in at all?